### PR TITLE
Add env template

### PR DESCRIPTION
**Pull request by Terris**
I'd like to be able to clone a redwood app and have an environment template ready to copy and paste over to my local .env file. This template might grow with a list of required keys, .etc as I add features like Stripe integration to my app and it'd be nice for contributors to have an updated env template of right out of the box too. Presently, only the project initiator (the one who ran yarn create-react-app) will have a .env with Redwood's required configs as this doesn't get checked in (appropriately so).

Thanks for considering :)

### CODE REVIEW

**Comment by peterp** [comment]
@Terris from what I understand we already supply a `.env` file that has some uncommented defaults, is `.env.example` a preferred standard?

**Comment by Terris** [comment]
@peterp - Yep, you do supply a .env file, but that doesn't get checked in when I git init and commit my project. Therefore, when I or another developer clones or forks this project, they do not have a template environment file to start from and won't have any idea what variables are necessary to even fire up the app.

Having a env.template (or env.example) with the required variables and example values and that always gets checked in would fix this. It's not really that big of a deal, but as my project configuration grows (stripe, analytics, test db, etc.) it will become tedious for collaborators to set up a new environment without some indication of what settings are required.

Am I making any sense? :)

**Comment by peterp** [comment]
Ah makes total sense, I like `.env.example`!

File: .env.template [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# DATABASE_URL=postgres://user:pass@postgreshost.com:5432/database_name
+# BINARY_TARGET=rhel-openssl-1.0.x
```
Comment (peterp) [inline .env.template:2]: I think this could be "native"

**Pull request closed without merging**